notifications/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from users.models import Member
+from django.utils import timezone
+from datetime import timedelta
+
+# TODO:
+# Consider merging notifications of the same type. How do we do this???
+# COULD MERGE BY URL
+
+# Create your models here.
+class Notification(models.Model):
+	member = models.ForeignKey(Member, related_name='notifications_owned', on_delete=models.CASCADE)
+	notify_type = models.CharField(max_length=32)
+	url = models.CharField(max_length=512)
+	content = models.TextField(max_length=8192)
+	unread = models.BooleanField()
```

Comment:
Nitpick: I think `is_unread` would be clearer

Reply:
Updated. Migrated just for you :P